src/Stratis.Bitcoin.Features.SmartContracts/Networks/SmartContractPosTest.cs

```diff
@@ -59,8 +57,8 @@ public SmartContractPosTest()
                 consensusFactory: consensusFactory,
                 consensusOptions: consensusOptions,
                 coinType: 1,
-                hashGenesisBlock: genesisBlock.Header.GetHash(),
-                subsidyHalvingInterval: 210000,
+                hashGenesisBlock: genesis.Header.GetHash(),
```

Comment:
`SmartContractPosRegTest.cs`, line 61 above does this - `this.Genesis.GetHash()`.  Just for consistency should it do the same here ?